### PR TITLE
feat: Launch copilot for CEA (js/ts)

### PR DESCRIPTION
**Pull request by QinghuiMeng-M**
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/29180531/

Launch copilot when CEA enabled

### CODE REVIEW

File: templates/js/custom-copilot-assistant-assistants-api/.vscode/launch.json.tpl [hunk 1]
```diff
@@ -75,6 +75,30 @@
...
+        {{#CEAEnabled}}
+        },
+        {
+            "name": "Launch in Microsoft 365 app (Edge)",
+            "type": "msedge",
+            "request": "launch",
+            "url": "https://www.office.com/chat",
+            "presentation": {
+                "group": "all",
+                "hidden": true
```
Comment (frankqianms) [inline templates/js/custom-copilot-assistant-assistants-api/.vscode/launch.json.tpl:87]: need to remove `hidden` settings to enable remote launch option, also need to add `extend to M365` action in teamsapp.yml. Please refer to latest changes in pr https://github.com/OfficeDev/teams-toolkit/pull/12593

File: templates/js/custom-copilot-assistant-assistants-api/.vscode/launch.json.tpl [hunk 2]
```diff
@@ -75,6 +75,30 @@
                 "order": 3
             },
             "internalConsoleOptions": "neverOpen",
+        {{#CEAEnabled}}
+        },
+        {
+            "name": "Launch Remote in Copilot (Edge)",
```
Comment (kuojianlu) [inline templates/js/custom-copilot-assistant-assistants-api/.vscode/launch.json.tpl:81]: Is the wording (in Copilot) confirmed with Junjie?

File: templates/js/custom-copilot-assistant-assistants-api/.vscode/launch.json.tpl [hunk 3]
```diff
@@ -75,6 +75,30 @@
...
+            "type": "msedge",
+            "request": "launch",
+            "url": "https://www.office.com/chat",
```
Comment (kuojianlu) [inline templates/js/custom-copilot-assistant-assistants-api/.vscode/launch.json.tpl:84]: Add account hint? See https://github.com/OfficeDev/teams-toolkit/blob/dev/templates/js/sso-tab-with-obo-flow/.vscode/launch.json#L63. 
You can test if the account hint is workable.

Comment (QinghuiMeng-M) [reply templates/js/custom-copilot-assistant-assistants-api/.vscode/launch.json.tpl]: ok, added

File: templates/js/custom-copilot-assistant-assistants-api/.vscode/launch.json.tpl (hunk 2 above)
Comment (QinghuiMeng-M) [reply templates/js/custom-copilot-assistant-assistants-api/.vscode/launch.json.tpl:81]: Frank has asked Junjie, waiting for his reply

**Review by kuojianlu — APPROVED** [review]

**Pull request merged**